desktop/desktop
# Add metrics for completing/aborting a merge after conflicts

⚠️ This PR has associated work: https://github.com/desktop/desktop/issues/5394#issue-350225825 ⚠️ 

Fixes #5394 

## How to test
1. Create a repo and produce some conflicts
1. Open that repo in Desktop (you should be notified that conflicts exist
1. Note the values of `mergeAbortedAfterConflictsCount ` and  `mergeSuccessAfterConflictsCount` from the database as _n_ and _m_ respectfully. 

### Abort
1. Open terminal and execute `git merge --abort`
1. Check the database: `mergeAbortedAfterConflictsCount ` should be **n + 1**

### Complete
1. Fix the conflicts
1. Make a commit
1. Check the database: `mergeSuccessAfterConflictsCount` should be **m + 1**
